### ISSUE TITLE
actix_raw not building

### ISSUE THREAD

**Issue by sebastienros** [TechEmpower/FrameworkBenchmarks#6049]
When I try to build actix-raw.dockerfile

```
[application]    Compiling v_escape v0.11.3
[application] error[E0432]: unresolved import `v_escape_derive::Escape`
[application]    --> /usr/local/cargo/registry/src/github.com-1ecc6299db9ec823/v_escape-0.11.3/src/lib.rs:109:5
[application]     |
[application] 109 | use v_escape_derive::Escape;
[application]     |     ^^^^^^^^^^^^^^^^^^^^^^^ no `Escape` in the root
[application]
[application] error: aborting due to previous error
[application]
[application] For more information about this error, try `rustc --explain E0432`.
[application] error: could not compile `v_escape`.
[application]
[application] To learn more, run the command again with --verbose.
[application] warning: build failed, waiting for other jobs to finish...
[application] error: build failed
[application] The command '/bin/sh -c RUSTFLAGS="-C target-cpu=native" cargo build --release' returned a non-zero code: 101
```

/cc robjtede

**Comment by billywhizz** [comment]
all the actix builds have been broken 12 september as far as i can see: https://tfb-status.techempower.com/results/301db83d-878d-433d-ae5e-1a25013e0b3c

**Comment by nbrady-techempower** [comment]
Looks solved by #6063
